docs/appendix.rst

```diff
@@ -524,8 +509,145 @@ To fix this, install the Python "wheel" package::
 
     pip install wheel
 
+
+this option should not be the preferred one.
+
+
+.. _`Issues with pywbem verson 1.7+, Urllib3 package version 2, and SSL`:
+
+Issues with pywbem verson 1.7+, Urllib3 package version 2, and SSL
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: pair; troubleshooting: OpenSSL
+.. index:: pair; SSL: OpenSSL
+.. index:: pair; UNSUPPORTED_PROTOCOL: OpenSSL
+
+Pywbem version 1.7 expanded the urllib3 package version requirement to include
+urllib version >= 2.0.
+
+The urlib3 package version requirements are in requirements.txt and are::
+
+      urllib3>=1.26.18; python_version >= '3.7'  (pywbem 1.7.0)
+
+      previously:
+
+      urllib3>=1.26.18,<2.0.0; python_version >= '3.7' (pywbem 1.6.x)
+
+The urllib3 Python package version 2.0 made a significant number of changes
+including the following list that can be potentially non-compatible:
+
+* Urllib3 >= 2.0 limits the minimum OpenSSL version >= 1.1.1.
+
+* Urllib3 >= 2.0 changes the default minimum TLS version to TLS 1.2 (previously was TLS 1.0)
+
+* Urllib3 >= 2.0 removed support for LibreSSL, WolfSSL and other SSL
+  implementations.  Multiple operating systems including: macOS and BSD use
+  LibreSSL as the TLS/SSL implementation. LibreSSL support however, was
+  restored to urllib3 in version 2.0.3.
+
+* Urllib3 >= 2.0 changed the minimum version for urllib3 version 2+ to Python 3.7.
+
+* Urllib3 >= 2.0 removed the default set of TLS ciphers, instead now urllib3
+  uses the list of ciphers configured by the system.
+
+Pywbem uses the requests dependent package which uses the urllib3 package which
+uses the pyopenssl to communicate with OpenSSL;
+neither pywbem nor requests pins the urllib3 package to versions below version 2.0.
+This is a function of the SSL implementation and pyopenssl.
+
+# TODO: This one bothers me.  It is the lib/ssl module which is a TLS/SSL wrapper
+# for socket objects that implemented PEP 644,  however, we never had any issues
+# with new minimum TLS version until 3.1.12 so the path through requests was
+# not impacted or we would have had to add urllib 2.0 in earlier tests.  It appears
+# that for the paths we use, it is requests, urllib3, pyOpenSSL packages and
+# OpenSSL version that are the issue.  I need to do another code review.
```

Comment:
I'm not sure I understand this commented out text. Is it not sufficient to state that Python 3.10 requires OpenSSL 1.1.1? That is in sync with what urllib3 2.0 requires.

Reply:
Removed the comment. Note that pyopenssl was actually used as the OpenSSL interface until the Python OpenSSL module was added to Python and that interface is an alternate (not recommended) in the urllib3 package.  Noted that the TLS 1.2 requirement comes from either Python 3.8 and/or urllib3 v2

The confusion is that this is a lot of parts/version all flying in loose formation with minimal communication.